src/NUnitEngine/nunit.engine/Services/AgentStatus.cs

```diff
@@ -29,11 +29,8 @@ namespace NUnit.Engine.Services
     /// </summary>
     public enum AgentStatus
     {
-        Unknown,
```

Comment:
I routinely use something like `Unknown` as the first enum member because that's what you get if you forget to initialize a field. If you drop it, then the default will become `Starting`.

Reply:
:+1: I was thinking about which becomes default as well.